hive-catalog-shade/pom.xml

```diff
@@ -285,49 +286,55 @@ under the License.
             </exclusions>
         </dependency>
         <dependency>
-            <groupId>org.apache.hive</groupId>
-            <artifactId>hive-standalone-metastore</artifactId>
-            <version>${hive.version}</version>
+            <groupId>org.apache.paimon</groupId>
+            <artifactId>paimon-hive-connector-3.1</artifactId>
+            <version>${paimon-hive-metastore.version}</version>
```

Comment:
`hive-standalone-metastore`  is the main dependency of all catalogs. I suggest that paimon-related dependencies move closer to it instead of replacing it, otherwise it will heavily rely on the hive version of paimon in the future.

Reply:
already edited